gazebo_ros2_control/include/gazebo_ros2_control/default_robot_hw_sim.h

```diff
@@ -0,0 +1,173 @@
+/*********************************************************************
```

Comment:
I think the ROS 2 style is to use the `.hpp` extension for headers, but I don't see that explained in the developer guide: https://index.ros.org/doc/ros2/Contributing/Code-Style-Language-Versions/#id1